answer/scanQRAck.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "http://json-schema.org/draft/2019-09/schema#",
+	"$id": "https://github.com/dedis/student20_pop/tree/proto-specs/answer/scanQRAck.json",
+	"title": "Match a custom JsonRpc 2.0 message answer",
```

Comment:
The title is the same as the file `answer/answer.json`. Some JsonValidators will display the title when debugging. You should probably correct it so that we can distinguish the two files :)